.github/workflows/ci.yml

```diff
@@ -54,8 +54,8 @@ jobs:
         try-scenario:
           - ember-lts-3.28
           - ember-lts-4.4
+          - ember-lts-4.8
           - ember-release
-          - ember-classic
           - embroider-safe
```

Comment:
Now that we have embroider-optimized passing, can you move it out of the experimental group and into the required list with embroider-safe here?

Reply:
good point! Done!